Crash report 5.5: Resources$NotFoundException on 4.1.1

On API 19 and below (4.1.1 emulator for instance):

* Go to site settings
* Tap "Start Over" -> crash

```
06-27 14:26:24.214 2141-2141/org.wordpress.android E/AndroidRuntime: FATAL EXCEPTION: main
android.content.res.Resources$NotFoundException: File res/drawable/gridicons_history.xml from drawable resource ID #0x7f020113
   at android.content.res.Resources.loadDrawable(Resources.java:1918)
   at android.content.res.Resources.getDrawable(Resources.java:659)
   at android.preference.Preference.onBindView(Preference.java:528)
   at org.wordpress.android.ui.prefs.WPPreference.onBindView(WPPreference.java:36)
   at org.wordpress.android.ui.prefs.WPStartOverPreference.onBindView(WPStartOverPreference.java:50)
   at android.preference.Preference.getView(Preference.java:453)
   at android.preference.PreferenceGroupAdapter.getView(PreferenceGroupAdapter.java:221)
   at android.widget.AbsListView.obtainView(AbsListView.java:2267)
   at android.widget.ListView.makeAndAddView(ListView.java:1769)
   at android.widget.ListView.fillDown(ListView.java:672)
   at android.widget.ListView.fillFromTop(ListView.java:733)
   at android.widget.ListView.layoutChildren(ListView.java:1622)
   at android.widget.AbsListView.onLayout(AbsListView.java:2102)
   at android.view.View.layout(View.java:13754)
   at android.view.ViewGroup.layout(ViewGroup.java:4362)
   at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1649)
   at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1507)
   at android.widget.LinearLayout.onLayout(LinearLayout.java:1420)
   at android.view.View.layout(View.java:13754)
   at android.view.ViewGroup.layout(ViewGroup.java:4362)
   at android.widget.FrameLayout.onLayout(FrameLayout.java:448)
   at android.view.View.layout(View.java:13754)
   at android.view.ViewGroup.layout(ViewGroup.java:4362)
   at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1649)
   at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1507)
   at android.widget.LinearLayout.onLayout(LinearLayout.java:1420)
   at android.view.View.layout(View.java:13754)
   at android.view.ViewGroup.layout(ViewGroup.java:4362)
   at android.widget.FrameLayout.onLayout(FrameLayout.java:448)
   at android.view.View.layout(View.java:13754)
   at android.view.ViewGroup.layout(ViewGroup.java:4362)
   at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:1866)
   at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1687)
   at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:998)
   at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:4212)
   at android.view.Choreographer$CallbackRecord.run(Choreographer.java:725)
   at android.view.Choreographer.doCallbacks(Choreographer.java:555)
   at android.view.Choreographer.doFrame(Choreographer.java:525)
   at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:711)
   at android.os.Handler.handleCallback(Handler.java:615)
   at android.os.Handler.dispatchMessage(Handler.java:92)
   at android.os.Looper.loop(Looper.java:137)
   at android.app.ActivityThread.main(ActivityThread.java:4745)
   at java.lang.reflect.Method.invokeNative(Native Method)
   at java.lang.reflect.Method.invoke(Method.java:511)
   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
   at dalvik.system.NativeStart.main(Native Method)
Caused by: org.xmlpull.v1.XmlPullParserException: Binary XML file line #1: invalid drawable tag vector
   at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:877)
   at android.graphics.drawable.Drawable.createFromXml(Drawable.java:818)
   at android.content.res.Resources.loadDrawable(Resources.java:1915)
   at android.content.res.Resources.getDrawable(Resources.java:659) 
   at android.preference.Preference.onBindView(Preference.java:528) 
   at org.wordpress.android.ui.prefs.WPPreference.onBindView(WPPreference.java:36) 
   at org.wordpress.android.ui.prefs.WPStartOverPreference.onBindView(WPStartOverPreference.java:50) 
   at android.preference.Preference.getView(Preference.java:453) 
   at android.preference.PreferenceGroupAdapter.getView(PreferenceGroupAdapter.java:221) 
   at android.widget.AbsListView.obtainView(AbsListView.java:2267) 
   at android.widget.ListView.makeAndAddView(ListView.java:1769) 
   at android.widget.ListView.fillDown(ListView.java:672) 
   at android.widget.ListView.fillFromTop(ListView.java:733) 
   at android.widget.ListView.layoutChildren(ListView.java:1622) 
   at android.widget.AbsListView.onLayout(AbsListView.java:2102) 
   at android.view.View.layout(View.java:13754) 
   at android.view.ViewGroup.layout(ViewGroup.java:4362) 
   at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1649) 
   at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1507) 
   at android.widget.LinearLayout.onLayout(LinearLayout.java:1420) 
   at android.view.View.layout(View.java:13754) 
   at android.view.ViewGroup.layout(ViewGroup.java:4362) 
   at android.widget.FrameLayout.onLayout(FrameLayout.java:448) 
   at android.view.View.layout(View.java:13754) 
   at android.view.ViewGroup.layout(ViewGroup.java:4362) 
   at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1649) 
   at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1507) 
   at android.widget.LinearLayout.onLayout(LinearLayout.java:1420) 
   at android.view.View.layout(View.java:13754) 
   at android.view.ViewGroup.layout(ViewGroup.java:4362) 
   at android.widget.FrameLayout.onLayout(FrameLayout.java:448) 
   at android.view.View.layout(View.java:13754) 
   at android.view.ViewGroup.layout(ViewGroup.java:4362) 
   at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:1866) 
   at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1687) 
   at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:998) 
   at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:4212) 
   at android.view.Choreographer$CallbackRecord.run(Choreographer.java:725) 
   at android.view.Choreographer.doCallbacks(Choreographer.java:555) 
   at android.view.Choreographer.doFrame(Choreographer.java:525) 
   at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:711) 
   at android.os.Handler.handleCallback(Handler.java:615) 
   at android.os.Handler.dispatchMessage(Handler.java:92) 
   at android.os.Looper.loop(Looper.java:137) 
   at android.app.ActivityThread.main(ActivityThread.java:4745) 
   at java.lang.reflect.Method.invokeNative(Native Method) 
   at java.lang.reflect.Method.invoke(Method.java:511) 
   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786) 
   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553) 
   at dalvik.system.NativeStart.main(Native Method) 

```

--- a/WordPress/src/main/java/org/wordpress/android/ui/prefs/WPStartOverPreference.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/prefs/WPStartOverPreference.java
@@ -31,8 +31,6 @@ public class WPStartOverPreference extends WPPreference {
     public WPStartOverPreference(Context context, AttributeSet attrs) {
         super(context, attrs);
 
-        mButtonTextColor = ContextCompat.getColor(context, R.color.black);
-
         TypedArray array = context.obtainStyledAttributes(attrs, R.styleable.WPStartOverPreference);
 
         for (int i = 0; i < array.getIndexCount(); ++i) {
@@ -40,7 +38,7 @@ public class WPStartOverPreference extends WPPreference {
             if (index == R.styleable.WPStartOverPreference_buttonText) {
                 mButtonText = array.getString(index);
             } else if (index == R.styleable.WPStartOverPreference_buttonTextColor) {
-                mButtonTextColor = array.getColor(index, ContextCompat.getColor(context, R.color.grey_dark));
+                mButtonTextColor = array.getColor(index, ContextCompat.getColor(context, R.color.black));
             } else if (index == R.styleable.WPStartOverPreference_buttonTextAllCaps) {
                 mButtonTextAllCaps = array.getBoolean(index, false);
             } else if (index == R.styleable.WPStartOverPreference_preficon) {

--- a/WordPress/src/main/java/org/wordpress/android/ui/prefs/WPStartOverPreference.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/prefs/WPStartOverPreference.java
@@ -2,11 +2,14 @@ package org.wordpress.android.ui.prefs;
 
 import android.content.Context;
 import android.content.res.TypedArray;
+import android.graphics.drawable.Drawable;
 import android.support.annotation.NonNull;
+import android.support.graphics.drawable.VectorDrawableCompat;
 import android.support.v4.content.ContextCompat;
 import android.util.AttributeSet;
 import android.view.View;
 import android.widget.Button;
+import android.widget.ImageView;
 import android.widget.TextView;
 
 import org.wordpress.android.R;
@@ -23,6 +26,7 @@ public class WPStartOverPreference extends WPPreference {
     private String mButtonText;
     private int mButtonTextColor;
     private boolean mButtonTextAllCaps;
+    private Drawable mPrefIcon;
 
     public WPStartOverPreference(Context context, AttributeSet attrs) {
         super(context, attrs);
@@ -39,6 +43,8 @@ public class WPStartOverPreference extends WPPreference {
                 mButtonTextColor = array.getColor(index, ContextCompat.getColor(context, R.color.grey_dark));
             } else if (index == R.styleable.WPStartOverPreference_buttonTextAllCaps) {
                 mButtonTextAllCaps = array.getBoolean(index, false);
+            } else if (index == R.styleable.WPStartOverPreference_preficon) {
+                mPrefIcon = VectorDrawableCompat.create(context.getResources(), array.getResourceId(index, 0), null);
             }
         }
 
@@ -48,6 +54,11 @@ public class WPStartOverPreference extends WPPreference {
     @Override
     protected void onBindView(@NonNull View view) {
         super.onBindView(view);
+
+        if (view.findViewById(R.id.pref_icon) != null) {
+            ImageView imageView = (ImageView) view.findViewById(R.id.pref_icon);
+            imageView.setImageDrawable(mPrefIcon);
+        }
 
         if (view.findViewById(R.id.button) != null) {
             final WPStartOverPreference wpStartOverPreference = this;

--- a/WordPress/src/main/res/layout/start_over_preference.xml
+++ b/WordPress/src/main/res/layout/start_over_preference.xml
@@ -18,7 +18,7 @@
         android:orientation="horizontal">
 
         <ImageView
-            android:id="@+android:id/icon"
+            android:id="@+id/pref_icon"
             android:layout_width="@dimen/start_over_icon_size"
             android:layout_height="@dimen/start_over_icon_size"
             android:layout_marginRight="@dimen/start_over_icon_margin_right"

--- a/WordPress/src/main/res/values/attrs.xml
+++ b/WordPress/src/main/res/values/attrs.xml
@@ -19,6 +19,7 @@
     </declare-styleable>
 
     <declare-styleable name="WPStartOverPreference">
+        <attr name="preficon" format="reference" />
         <attr name="buttonText" format="string" />
         <attr name="buttonTextColor" format="reference|color" />
         <attr name="buttonTextAllCaps" format="boolean" />

--- a/WordPress/src/main/res/xml/site_settings.xml
+++ b/WordPress/src/main/res/xml/site_settings.xml
@@ -293,11 +293,11 @@
             <org.wordpress.android.ui.prefs.WPStartOverPreference
                 android:id="@+id/pref_start_over"
                 android:key="@string/pref_key_site_start_over"
-                android:icon="@drawable/gridicons_history"
                 android:title="@string/let_us_help"
                 android:summary="@string/start_over_text"
                 android:widgetLayout="@layout/start_over_preference_button"
                 android:layout="@layout/start_over_preference"
+                app:preficon="@drawable/gridicons_history"
                 app:longClickHint="@string/site_settings_start_over_hint"
                 app:buttonText="@string/contact_support"
                 app:buttonTextAllCaps="true" />

--- a/WordPress/src/main/res/xml/site_settings.xml
+++ b/WordPress/src/main/res/xml/site_settings.xml
@@ -300,6 +300,7 @@
                 app:preficon="@drawable/gridicons_history"
                 app:longClickHint="@string/site_settings_start_over_hint"
                 app:buttonText="@string/contact_support"
+                app:buttonTextColor="@color/grey_dark"
                 app:buttonTextAllCaps="true" />
 
         </PreferenceScreen>

